command/command.c

```diff
@@ -200,6 +219,23 @@ settings_command_changed (GSettings *settings, gchar *key, CommandApplet *comman
         command_applet->command = g_strdup ("");
 }
 
+/* GSettings signal callbacks */
+static void
+settings_command_launcher_changed (GSettings *settings, gchar *key, CommandApplet *command_applet)
+{
+    gchar *command_launcher;
+
+    command_launcher = g_settings_get_string (command_applet->settings, COMMAND_LAUNCHER_KEY);
+
+    if (command_applet->command_launcher)
+        g_free (command_applet->command_launcher);
```

Comment:
`g_free` is NULL safe. :) Also `g_settings_get_string` does not return `NULL` so all this can be simplified to:
```
g_free (command_applet->command_launcher);
command_applet->command_launcher = command_launcher;
```